src/context/AuthContext.js

```diff
@@ -0,0 +1,44 @@
+import { createContext, useContext, useEffect } from "react";
+import {
+    createUserWithEmailAndPassword,
+    signInWithEmailAndPassword,
+    // signOut,
+    // onAuthStateChanged
+} from "firebase/auth";
+import { auth } from "../utils/firebase";
+import { useState } from "react";
+
+const UserContext = createContext();
+
+export const AuthContextProvider = ({children}) => {
```

Comment:
do u need this context when using firebase? can u access user object from firebase directly?